src/app/home/footer/footer.component.css

```diff
@@ -0,0 +1,15 @@
+/* original font-size=16px, 14px looks better*/
+.font-family-semibold{
+    font-family: 'Source Sans Pro', sans-serif;
+    font-weight: 600 ;
+    font-size: 14px;
+    color: #ffff;
+}
+.opacity{
```

Comment:
Extra space needed between the class name and the curly bracket. Also extra space needed between this class and the class above.